datavault-webapp/src/main/java/org/datavaultplatform/webapp/controllers/admin/AdminRolesController.java

```diff
@@ -177,6 +199,68 @@ public ResponseEntity deleteRole(@RequestParam("id") long roleId) {
         return ResponseEntity.ok().build();
     }
 
+    @PostMapping("/admin/roles/op")
+    public ResponseEntity addIsAdmin(Principal principal, @RequestParam("op-id") String userId) {
+
+        logger.debug("Preparing to add user ID={} to IS ADMIN role", userId);
+        RoleModel role = restService.getIsAdmin();
+        User user = restService.getUser(userId);
+
+        Optional<RoleAssignment> roleAssignment = restService.
```

Comment:
This is an odd use of optional since you don't consume the wrapped `RoleAssignment`. It'd be a bit cleaner to use `anyMatch()` on the stream instead of `.filter().collect()`.

Reply:
Yeah, I thought I might need the variable later, but I don't. Changed.